packages/@aws-cdk/aws-chatbot/README.md

```diff
@@ -27,25 +27,28 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
 
 ```ts
 import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as lambda from '@aws-cdk/aws-lambda';
 
 const slackChannel = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel', {
   slackChannelConfigurationName: 'YOUR_CHANNEL_NAME',
   slackWorkspaceId: 'YOUR_SLACK_WORKSPACE_ID',
   slackChannelId: 'YOUR_SLACK_CHANNEL_ID',
 });
 
-slackChannel.addLambdaInvokeCommandPermissions();
-slackChannel.addNotificationPermissions();
-slackChannel.addSupportCommandPermissions();
-slackChannel.addReadOnlyCommandPermissions();
-
 slackChannel.addToPrincipalPolicy(new iam.PolicyStatement({
   effect: iam.Effect.ALLOW,
   actions: [
     's3:GetObject',
   ],
   resources: ['arn:aws:s3:::abc/xyz/123.txt'],
 }));
+
+const lambdaFn = new lambda.Function(this, 'MyChatbotHandler', {
+  code: lambda.Code.fromAsset('./code'),
+  runtime: lambda.Runtime.NODE_JS_14_X,
+  handler: 'FILE_NAME.handler',
+});
+lambdaFn.grantInvoke(slackChannel.role)
```

Comment:
Actually I'm not sure if this is needed here. I misinterpreted the point of the previous calls I believe. I'm not sure if `addLambdaInvokeCommandPermissions` is actually meant to grant access to a single specific lambda function. These aren't managed policies but just templates that the chatbot console will create a role with.

Perhaps we should just remove the usage of the methods that don't exist for now.

Reply:
Yuh, that method is for granting permissions in order to access a particular lambda function. Should we remove this?

Reply:
yeah lets remove this and the only changes from the original will be removing the calls to the deprecated methods. I do think users should be granting access to specific lambdas, cloudwatch metrics, etc and not using `*` resources. This however should be demonstrated in a more specific example.